assets/css/extended/homepage-daypage.css

```
@@ -0,0 +1,1594 @@
+/* ============================================================
+   Homepage — DayPage Design System Redesign
+   Space Grotesk + Inter + JetBrains Mono
+   Warm amber editorial aesthetic
+   ============================================================ */
+
+@import url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@500;600;700&family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500;700&display=swap');
```

Stylelint (17.9.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@500;600;700&family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@500;600;700&family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500;700&display=swap'" (import-notation)

(import-notation)